config/environments/production.rb

```diff
@@ -127,27 +127,8 @@
   protocol = ENV["APP_PROTOCOL"] || "http://"
 
   config.action_mailer.delivery_method = :smtp
-  config.action_mailer.perform_deliveries =  ENV["SMTP_PASWORD"].present? || ENV["SENDGRID_API_KEY"].present?
+  config.action_mailer.perform_deliveries = true
```

Comment:
**blocking**: this should default to `false`, as it's determined at runtime

Reply:
```suggestion
  config.action_mailer.perform_deliveries = false
```